crates/ra_analysis/src/imp.rs

```diff
@@ -184,6 +184,28 @@ impl AnalysisImpl {
         };
         Ok(query.search(&buf))
     }
+
+    pub(crate) fn module_path(&self, position: FilePosition) -> Cancelable<String> {
+        let descr = match source_binder::module_from_position(&*self.db, position)? {
+            None => return Ok("".to_string()),
+            Some(it) => it,
+        };
+        let name = match descr.name() {
+            None => return Ok("".to_string()),
+            Some(it) => it.to_string(),
+        };
+
+        let modules = descr.path_to_root();
+
+        let path = modules
+            .into_iter()
+            .filter_map(|s| s.name())
+            .skip(1)
```

Comment:
I'm sure it's there for a reason, but a comment explaining what `skip(1)` does in this case would be useful.